nucypher/policy/conditions/lingo.py

```diff
@@ -235,6 +237,19 @@ def __init__(self, comparator: str, value: Any, index: int = None):
             )
 
         if not is_context_variable(value):
+            # adjust stored value to be JSON serializable
+            if isinstance(value, (tuple, set)):
+                value = list(value)
+            if isinstance(value, bytes):
+                value = HexBytes(value).hex()
+
+            try:
+                json.dumps(value)
+            except Exception:
```

Comment:
Why are we catching anything other than `TypeError` here?  Presumably we don't want to catch, eg, `OverflowError`?

Is the reason we're catching this in the first place simply that we need to raise `InvalidExpression` for the caller's sake?

Reply:
Agree

```suggestion
            except TypeError
            ```

Reply:
✔️  - Indeed. This was an oversight